instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './CommentSection.css';
+import Comment from './Comment/Comment';
+import moment from 'moment';
+
+class Comments extends React.Component{
+  constructor(props){
+    super(props)
+    this.state ={
+      comments: props.comments,
+      comment: ''
+    }
+  }
+
+  commentHandleChange = e => {
+    this.setState({comment: e.target.value})
+  }
+
+  addNewComment = (e) => {
+    e.preventDefault();
+    const comments = this.state.comments.slice();
+    const newComment = {text: this.state.comment, username: 'winniesongii'};
+    if (this.state.comment !== ''){
+      comments.push(newComment);
+      this.setState({comments, comment: ''});
```

Comment:
Good work on this bit of code. Remember the spread operator can replace slice() and may be easier to read. I like the field reset in your setState.